components/dialog/test/dialog.visual-diff.js

```diff
@@ -46,23 +46,30 @@ describe('d2l-dialog', () => {
 						await page.setViewport({ width: info.viewport.width, height: info.viewport.height, deviceScaleFactor: 2 });
 					});
 
-					it('opened', async function() {
-						if (info.category === 'short-narrow') return; // TODO: remove this; skipping for now due to flaking
-						await helper.open(page, '#dialog');
-						await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
+					it('opened', function() {
+						return Promise.all([
```

Comment:
This parallelizes these tasks, but the screenshot task needs to happen after.  This might luckily work but it doesn't look like it would always work.

Reply:
Ah I was under the impression that the tasks are happening in serial, though upon further searching it seems a bit vague and unknown. It has been working consistently for these cases for several runs but I'll see about reorganizing to be more sure.